src/core/lib/security/transport/client_auth_filter.cc

```diff
@@ -157,7 +160,15 @@ ArenaPromise<absl::StatusOr<CallArgs>> ClientAuthFilter::GetCallCredsMetadata(
 
   auto client_initial_metadata = std::move(call_args.client_initial_metadata);
   return TrySeq(
-      creds->GetRequestMetadata(std::move(client_initial_metadata), &args_),
+      Seq(creds->GetRequestMetadata(std::move(client_initial_metadata), &args_),
```

Comment:
I see, so we need to check all filters where a per-channel logic is affecting an RPC's status. Have you checked all filters? 
Is there a place where we can document this for new filter writers?

Reply:
The goal here is not to check all filters.  The [gRFC](https://github.com/grpc/proposal/blob/master/A54-restrict-control-plane-status-codes.md#proposal) specifies 4 individual cases that need to be covered, and those are the ones covered in this PR.